kupfer/plugin/websearch.py

```diff
@@ -18,7 +18,7 @@
 from kupfer.objects import TextLeaf
 from kupfer import utils, config
 
-from kupfer.plugin import firefox_support
+from kupfer.plugin import firefox
```

Comment:
importing other plugins is not allowed, the _support modules are used for sharing code between them. However the more high level issues will be discussed first naturally.